tslint.json

```diff
@@ -0,0 +1,8 @@
+{
+  "extends": "gts/tslint.json",
+  "linterOptions": {
+    "exclude": [
+      "**/*.json"
+    ]
+  }
+}
```

Comment:
I cannot add an excluding rule when run `gts check`.
Here is my snippet code for exception lint:
`"rules": {
    "no-any": false
  }`

Could you try it to see if we can add exception lint with gts?

Reply:
@minh-rakuten I think the base rules look fine. Let's check if we need to customize and the background behind. + Let's do it beyond the scope of this PR.